petagene_compress.sh

```diff
@@ -4,14 +4,10 @@
 #$ -l excl=1
 #$ -pe mpi 40
 
-export LD_PRELOAD=/usr/lib/petalink.so
-export PETASUITE_REFPATH=/seqstore/software/petagene/corpus:/opt/petagene/petasuite/species
-
-
 OUTPUTDIR=$(echo "$1")
 
-find $OUTPUTDIR -name "*_REALIGNED.bam" -exec petasuite -c -X --numthreads 40 -m bqfilt --validate full {} \;
-find $OUTPUTDIR -name "*_realignedTNscope.bam" -exec petasuite -c -X --numthreads 40 -m bqfilt --validate full {} \;
-
 find $OUTPUTDIR -name "*.bam" -not -name "*_REALIGNED.bam" -not -name "*_realignedTNscope.bam" -exec rm {} \;
 find $OUTPUTDIR -name "*.bam.bai" -not -name "*_REALIGNED.bam.bai" -not -name "*_realignedTNscope.bam.bai" -exec rm {} \;
+
+find $OUTPUTDIR -name "*_REALIGNED.bam" -exec petasuite -c -X --numthreads 40 -m bqfilt --validate full {} \;
```

Comment:
Feels like it should be possible to have these paths without using unix find? 😆

Reply:
probably. will look more at this when I have to move the bam file to IGV before compressing it